tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,46 @@
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    for item_name, item_count in inventory.items():
+        print(item_count, item_name)
+        item_total += item_count
+    print("Total number of items: " + str(item_total))
+    if item_total >= 80:
```

Comment:
I think it would be good to introduce constants here.

eg. 
`MAX_INVENTORY_CAP = 80`
`HEAVY_WEIGHT_THRESHOLD = 70`
`LIGHT_WEIGHT_THRESHOLD=60`

Feel free to use other naming names if u come up with better ones :D
Also, please double-check the numbers I've provided here, that's just a proposition. 

And of course, consts should be moved to the top of the file.